examples/long-term-schedule-transaction.js

```diff
@@ -0,0 +1,198 @@
+// LongTermScheduledTransactionExample.js
```

Comment:
Is it needed?

If you'd like to provide a description for the example, you can do so using
 
```
/**
* @description text here
*/
```

before the function declaration